### PR TITLE
grayskull recipe for podaac-data-subscriber

### PR DESCRIPTION
**Pull request by kthyng**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Review by beenje — CHANGES_REQUESTED** [review]
Windows build failed with `Noarch package contains binary script: podaac-data-subscriber.exe`. I think it might be because of the missing `entry_points` definition in build. Add them and I think it should work.

File: recipes/podaac-data-subscriber/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/podaac-data-subscriber-{{ version }}.tar.gz
+  sha256: d341d3a09c13eb59eacc5545ac60268ab46139d74947335bd87e1aef0e7661a1
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```
[the remark this answers is not in the document]
Comment (kthyng) [reply recipes/podaac-data-subscriber/meta.yaml:15]: I would not have figured this one out, thanks.

File: recipes/podaac-data-subscriber/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - subscriber
+  commands:
+    - pip check
```
[the remark this answers is not in the document]
Comment (kthyng) [reply recipes/podaac-data-subscriber/meta.yaml:29]: Or this...

**Comment by kthyng** [comment]
@beenje Thanks for your help! It looks like I got it working with your changes. Is there anything else I can do on my end?

**Comment by beenje** [comment]
Looks good to me now!

**Pull request merged**